src/excalidraw-app/collab/CollabWrapper.tsx

```diff
@@ -382,16 +280,6 @@ class CollabWrapper extends PureComponent<Props, State> {
         activeRoomLink: window.location.href,
         isLoading: opts.showLoadingState ? true : this.state.isLoading,
       });
-
-      try {
-        const elements = await loadFromFirebase(roomID, roomKey);
-        if (elements) {
-          this.handleRemoteSceneUpdate(elements, { initFromSnapshot: true });
-        }
-      } catch (e) {
-        // log the error and move on. other peers will sync us the scene.
-        console.error(e);
-      }
```

Comment:
moved to `ExcalidrawApp` because this part serves to init the scene (hence be an `initialData`).